flink-dist/src/main/flink-bin/bin/flink-daemon.sh

```diff
@@ -88,15 +88,6 @@ out="${FLINK_LOG_PREFIX}.out"
 
 log_setting=("-Dlog.file=${log}" "-Dlog4j.configuration=file:${FLINK_CONF_DIR}/log4j.properties" "-Dlog4j.configurationFile=file:${FLINK_CONF_DIR}/log4j.properties" "-Dlogback.configurationFile=file:${FLINK_CONF_DIR}/logback.xml")
 
-JAVA_VERSION=$(${JAVA_RUN} -version 2>&1 | sed 's/.*version "\(.*\)\.\(.*\)\..*"/\1\2/; 1q')
```

Comment:
Why is this being removed? The same code exists in flink-console.sh.

Reply:
Damn, I need to work on my `git grep` skills 😄 

I removed it because the java version is used to check if the version is less than 1.8. Since it can never be less than 1.8, we don't need this check (anymore).